app/javascript/elm/tests/MainTests.elm

```diff
@@ -22,6 +23,26 @@ import Test.Html.Selector as Slc
 import TimeRange
 
 
+heatMapThresholdsWithMinimum : Int -> HeatMapThresholds
+heatMapThresholdsWithMinimum value =
+    { threshold1 = { value = value, default = 1 }
```

Comment:
what if value is higher than one or all of the other thresholds?

Reply:
I guess for the purpose of the tests I'm just looking at that value regardless of the others. It's a good point though. We could have a smart constructor that does the checking and returns a maybe in case we needed to have that in the production code. Something like
```
build : Maybe Heat
build val =
  if isOk val then
    Just ..
  else
    Nothing
```